data/transactions/logic/opcodes.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // LogicVersion defines default assembler and max eval versions
-const LogicVersion = 12
+const LogicVersion = 13
```

Comment:
Let's leave this alone in this PR.  We would bump it at release time when we need to move falcon_verify and sumhash512 out of the way.